### DISCUSSION TITLE
+R FRC training

### DISCUSSION THREAD

**Discussion by Kiumbra** [nirgoren/Eddienput#19] [Technical support]
Is there a specific way to set up the beep to train frcs or do I set a beep after a delay and bind the script to start from my fightstick button?

**Comment by KuukuuOmoi** [comment]
try a "+ BEEP " to the command you want the beep on

**Comment by hellshakeyan0** [comment]
From what I understand, the idea is to set a bot to do the FRC, watch it, listen for the beep, and learn it that way.  It didn't help me much more than the built in FRC indicator (which isn't saying much lol) but it's there.
